gordo_components/cli/cli.py

```diff
@@ -188,11 +188,13 @@ def build(
         config wherever there is a jinja variable with the key.
     model_location_file: str/path
         Path to a file to open and write the location of the serialized model to.
-    cv_mode: str
-        String which enables three different modes:
-        * cross_val_only: Only perform cross validation
-        * build_only: Skip cross validation and only build the model
-        * full_build: Cross validation and full build of the model, default value
+    evaluation_config: dict
+        Dict of parameters which are exposed to build_model.
+            - cv_mode: str
+                String which enables three different modes:
+                * cross_val_only: Only perform cross validation
+                * build_only: Skip cross validation and only build the model
+                * full_build: Cross validation and full build of the model, default value
```

Comment:
Maybe here too, refer to the 'master' doc of this parameter in `build_model` only that it would be in its string representation form?

Reply:
I am a bit uncertain about having the 'master' doc in the ```build_model```, as the ```cli``` can be used "without" knowing about the ```build_model```, and for a user to have to "dig" into the code to find it might not be the best. Thoughts? @milesgranger